Python/libraries/recognizers-date-time/recognizers_date_time/date_time/french/datetimeperiod_parser_config.py

```diff
@@ -64,6 +64,7 @@ def __init__(self, config: BaseDateParserConfiguration):
             FrenchDateTime.RelativeTimeUnitRegex)
         self._rest_of_date_time_regex = RegExpUtility.get_safe_reg_exp(
             FrenchDateTime.RestOfDateTimeRegex)
+        self._time_zone_parser = BaseTimeZoneParser()
```

Comment:
This is not correct. This is C# initialization of this prop

`TimeZoneParser = config.TimeZoneParser;`